packages/itwin/tree-widget/src/components/TreeFilteringState.ts

```diff
@@ -3,8 +3,9 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import React, { useState } from "react";
-import type { IPresentationTreeDataProvider } from "@itwin/presentation-components";
+import { IPresentationTreeDataProvider } from "@itwin/presentation-components";
```

Comment:
was it intentional to remove the `type` import?

Reply:
Yes, I did that for consistency with other imports in the package. 

I guess I don't see the value of using these `import type` imports since I have never seen any issues with simple `import`, so I just choose to use the latter everywhere. Did Design Review have any problems with that?

Reply:
@grigasp no its no issue, but I would recommend we make it the standard for imports that can be done as type since that will alleviate the import from getting pulled in the javascript, which an isolated case like this is unlikely to have any impact but if done consistently it can

Reply:
I wonder if there's any lint rule that would help us with that, otherwise it would certainly be very hard to not make a mess of those imports, especially when auto-importing. 

In any case, if this is not a problem at the moment, let's get this in and we can get back to **consistently** using `import type` afterwards.

Reply:
Filed a [backlog item](https://github.com/iTwin/viewer-components-react/issues/470) for this and closing the thread.